packages/devtools_shared/lib/src/extensions/extension_manager.dart

```diff
@@ -72,6 +72,10 @@ class ExtensionsManager {
     // Find all runtime extensions for [rootFileUriString], if non-null and
     // non-empty.
     if (rootFileUriString != null && rootFileUriString.isNotEmpty) {
+      logs.add(
+        'ExtensionsManager.serveAvailableExtensions adding extensions for app '
+        'root.',
+      );
```

Comment:
Is there value in including the root in the log like the entry below does?

Reply:
The log above already has `rootFileUriString` in it. These extra logs are just to help signal which types of roots we are actually trying to add (ones from the app root or ones from DTD roots.